front/components/actions/mcp/CreateMCPServerDialog.tsx

```diff
@@ -317,7 +317,7 @@ export function CreateMCPServerDialog({
                 : "Add MCP Server"}
           </DialogTitle>
         </DialogHeader>
-        <DialogContainer>
+        <DialogContainer className="max-h-[80vh] space-y-4 overflow-y-auto">
```

Comment:
Makes sure the diaglog container is scrollable if there's a big form + adding space between sections for visual clarity